software/bsg_manycore_lib/bsg_shared_mem.hpp

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
```

Comment:
Use ifdef, not #once (which is historically gcc specific)

Reply:
I’m pretty sure clang supports once?

Reply:
It does (since 2014). But historically it started with GCC.